hash_practice/exercises.py

```diff
@@ -1,19 +1,52 @@
 
+def create_anagram_key(string):
+    return ''.join(sorted(string))
+
+
 def grouped_anagrams(strings):
     """ This method will return an array of arrays.
         Each subarray will have strings which are anagrams of each other
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(n*m)

```

Comment:
Yes! Where n is the length of `strings` and m is the length of the longest word in `strings`. However, if we assume each word in `strings` is a valid English word, since those are all of a finite length, we can reduce this to say the solution is O(n)